applications/connectivity_bridge/src/modules/uart_handler.c

```diff
@@ -70,7 +52,10 @@ BUILD_ASSERT((sizeof(struct uart_rx_buf) % UART_SLAB_ALIGNMENT) == 0);
 
 K_MEM_SLAB_DEFINE(uart_rx_slab, UART_SLAB_BLOCK_SIZE, UART_SLAB_BLOCK_COUNT, UART_SLAB_ALIGNMENT);
 
-static const struct device *devices[UART_DEVICE_COUNT];
+static const struct device *devices[UART_DEVICE_COUNT] = {
```

Comment:
How about:
```suggestion
static const struct device *devices[] = {
```
and defining `UART_DEVICE_COUNT` as `ARRAY_SIZE(devices)`?